webapp/components/interop-data.js

```diff
@@ -579,5 +579,408 @@ export const interopData = {
+     * More information on results generation at
+     * https://github.com/web-platform-tests/results-analysis
+    **/
+    'csv_url': 'https://raw.githubusercontent.com/web-platform-tests/results-analysis/gh-pages/data/interop-2024/interop-2024-{stable|experimental}-v2.csv',
+    'summary_feature_name': 'summary',
+    'issue_url': 'https://github.com/web-platform-tests/interop/issues/new',
+    'focus_areas': {
+      'interop-2021-aspect-ratio': {
+        'description': 'Aspect Ratio',
+        'mdn': 'https://developer.mozilla.org/docs/Web/CSS/aspect-ratio',
+        'spec': 'https://drafts.csswg.org/css-sizing/#aspect-ratio',
+        'tests': '/results/?label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2021-aspect-ratio',
+        'countsTowardScore': false
+      },
+      'interop-2021-position-sticky': {
+        'description': 'Sticky Positioning',
+        'mdn': 'https://developer.mozilla.org/docs/Web/CSS/position',
+        'spec': 'https://drafts.csswg.org/css-position/#position-property',
+        'tests': '/results/css/css-position/sticky?label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2021-position-sticky',
+        'countsTowardScore': false
+      },
+      'interop-2022-cascade': {
+        'description': 'Cascade Layers',
+        'mdn': 'https://developer.mozilla.org/docs/Web/CSS/@layer',
+        'spec': 'https://drafts.csswg.org/css-cascade/#layering',
+        'tests': '/results/css/css-cascade?label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-cascade',
+        'countsTowardScore': false
+      },
+      'interop-2022-dialog': {
+        'description': 'Dialog Element',
+        'mdn': 'https://developer.mozilla.org/docs/Web/HTML/Element/dialog',
+        'spec': 'https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element',
+        'tests': '/results/?label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-dialog',
+        'countsTowardScore': false
+      },
+      'interop-2022-text': {
+        'description': 'Typography and Encodings',
+        'mdn': 'https://developer.mozilla.org/docs/Web/CSS/length#relative_length_units_based_on_viewport',
+        'spec': '',
+        'tests': '/results/?label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-text',
+        'countsTowardScore': false
+      },
+      'interop-2022-viewport': {
+        'description': 'Viewport Units',
+        'mdn': '',
+        'spec': 'https://drafts.csswg.org/css-values/#viewport-relative-units',
+        'tests': '/results/css/css-values?label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-viewport',
+        'countsTowardScore': false
+      },
+      'interop-2022-webcompat': {
+        'description': 'Web Compat',
```

Comment:
As in https://github.com/web-platform-tests/wpt.fyi/pull/3653:

```suggestion
        'description': 'Web Compat 2022',
```